### PR TITLE
test to show slow purges

### PR DESCRIPTION
**Pull request by ripienaar**
Signed-off-by: R.I.Pienaar <rip@devco.net>

/cc @nats-io/core

### CODE REVIEW

File: server/norace_test.go [hunk 1]
```diff
@@ -1756,6 +1757,89 @@ func TestNoRaceJetStreamClusterSourcesMuxd(t *testing.T) {
...
+		}
+	}
+
+	s := RunBasicJetStreamServer()
+	defer s.Shutdown()
+
+	nc, js := jsClientConnect(t, s)
+	defer nc.Close()
+
+	cfg := StreamConfig{
```
Comment (derekcollison) [inline server/norace_test.go:1776]: Can be normal js.AddStream. Check jetstream_cluster_test.go

File: server/norace_test.go [hunk 2]
```diff
@@ -1756,6 +1757,89 @@ func TestNoRaceJetStreamClusterSourcesMuxd(t *testing.T) {
...
+	cerr(t, err)
+
+	_, err = nc.Request(fmt.Sprintf(JSApiStreamCreateT, cfg.Name), req, time.Second)
+	cerr(t, err)
+
+	si, err := js.StreamInfo("KV")
+	cerr(t, err)
+
+	// 10kb messages spread over 1000 subjects
+	body := make([]byte, 101024)
```
Comment (derekcollison) [inline server/norace_test.go:1791]: This supposed to be 10*1024?

File: server/norace_test.go [hunk 3]
```diff
@@ -1756,6 +1757,89 @@ func TestNoRaceJetStreamClusterSourcesMuxd(t *testing.T) {
...
+		err := nc.Publish(fmt.Sprintf("kv.%d", i%1000), body)
+		cerr(t, err)
+	}
+	si, err = js.StreamInfo("KV")
+	cerr(t, err)
+	if si == nil || si.Config.Name != "KV" {
+		t.Fatalf("StreamInfo is not correct %+v", si)
+	}
+
+	func() {
```
Comment (derekcollison) [inline server/norace_test.go:1802]: Can use checkFor, see jetstream_cluster_test.go for some examples.

**Review by derekcollison — CHANGES_REQUESTED** [review]

File: server/norace_test.go (hunk 1 above)
Comment (ripienaar) [reply server/norace_test.go:1776]: ah, I initially had a per subject bit in there so did it this way but removed it.

File: server/norace_test.go (hunk 2 above)
Comment (ripienaar) [reply server/norace_test.go:1791]: something like that :P I probably just fiddled numbers till it broke, should be 100kb

**Review by derekcollison — APPROVED** [review]
LGTM

**Pull request merged**